randovania/games/dread/presets/starter_preset.rdvpreset

```diff
@@ -167,6 +167,7 @@
         "hanubia_shortcut_no_grapple": true,
         "hanubia_easier_path_to_itorash": true,
         "x_starts_released": false,
+		"allow_highly_dangerous_logic": false,
```

Comment:
You need to run `randovania development refresh-presets` instead of manually updating presets.

Reply:
I assume this is a simple command like the one for prepare_virtual_env?

Reply:
i can't seem to find where the refresh-presets is and i dont see any development folders anywhere

Reply:
This has been handled now thanks to dunc